### ISSUE TITLE
can't select custom prompt in CodeAssist

### ISSUE THREAD

**Issue by pinkyo** [SenseTime-Copilot/Raccoon#82]
Type: <b>Bug</b>

1. configure a custon prompt
2. select some codes
3. select custom prompt in code assitant panel (has issue).

Extension version: 0.75.1
VS Code version: Code 1.90.2 (5437499feb04f7a586f677b155b039bc2b3669eb, 2024-06-18T22:37:41.291Z)
OS version: Darwin x64 23.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|5, 6, 6|
|Memory (System)|16.00GB (0.82GB free)|
|Process Argv|--enable-proposed-api Tencent-Cloud.coding-copilot --crash-reporter-id 50d86310-0f2d-4d05-a5be-c3640014b1e0|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
724cj586:31013169
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
2f103344:31071589
legacy_priority:31082724

```

</details>

**Comment by pinkyo** [comment]
![image](https://github.com/SenseTime-Copilot/Raccoon/assets/32171700/69e7fee4-dc78-423d-ae72-010b2ace05c8)

**Comment by pinkyo** [comment]
![image](https://github.com/SenseTime-Copilot/Raccoon/assets/32171700/d04e89f2-6012-4d26-bbed-7e74848cb899)

**Comment by todd-xander** [comment]
@pinkyo Thx for your feedback. Since we could not reproduce this issue, is there any more infromation?

**Comment by pinkyo** [comment]
You can test with my custom prompt content. In my test:
don't work:
```
Add comments and docstring in english and golang style to {{code}}
Caution: don't modify code.
```
work:
`Add comments and docstring in english and golang style to {{code}}`

**Comment by todd-xander** [comment]
> You can test with my custom prompt content. In my test: don't work:
> 
> ```
> Add comments and docstring in english and golang style to {{code}}
> Caution: don't modify code.
> ```
> 
> work: `Add comments and docstring in english and golang style to {{code}}`

Thx so much, will fix it soon

**Comment by todd-xander** [comment]
@pinkyo We have fixed this issue on v0.76.0, and has been publisher to marketplace, try it.
